fuel-vm/src/checked_transaction.rs

```diff
@@ -880,22 +900,21 @@ mod tests {
     // input
     #[quickcheck]
     fn max_fee_message_input(
```

Comment:
The purpose of this test to test different `gas_price` as well, could you return it back, please?=)